configs/scannetpp/insseg-pointgroup-v1m1-0-spunet-base.py

```diff
@@ -0,0 +1,387 @@
+    ),
+    backbone_out_channels=64,
+    semantic_num_classes=num_classes,
+    semantic_ignore_index=-1,
+    segment_ignore_index=segment_ignore_index,
+    instance_ignore_index=-1,
+    cluster_thresh=1.5,
+    cluster_closed_points=300,
+    cluster_propose_points=100,
+    cluster_min_points=50,
+)
+
+# scheduler settings
+epoch = 800
+optimizer = dict(type="SGD", lr=0.1, momentum=0.9, weight_decay=0.0001, nesterov=True)
+scheduler = dict(type="PolyLR")
+
+# dataset settings
+dataset_type = "ScanNetPPDataset"
+# data_root = "data/scannetpp"
+data_root = "./raw_dataset/scannetpp_v2_sgi"
+
+data = dict(
+    num_classes=num_classes,
+    ignore_index=-1,
+    names=class_names,
+    train=dict(
+        type=dataset_type,
+        split="train",
+        data_root=data_root,
+        transform=[
+            dict(type="CenterShift", apply_z=True),
+            dict(
+                type="RandomDropout", dropout_ratio=0.2, dropout_application_ratio=0.5
+            ),
+            # dict(type="RandomRotateTargetAngle", angle=(1/2, 1, 3/2), center=[0, 0, 0], axis='z', p=0.75),
+            dict(type="RandomRotate", angle=[-1, 1], axis="z", center=[0, 0, 0], p=0.5),
+            dict(type="RandomRotate", angle=[-1 / 64, 1 / 64], axis="x", p=0.5),
+            dict(type="RandomRotate", angle=[-1 / 64, 1 / 64], axis="y", p=0.5),
+            dict(type="RandomScale", scale=[0.9, 1.1]),
+            # dict(type="RandomShift", shift=[0.2, 0.2, 0.2]),
+            dict(type="RandomFlip", p=0.5),
+            dict(type="RandomJitter", sigma=0.005, clip=0.02),
+            dict(type="ElasticDistortion", distortion_params=[[0.2, 0.4], [0.8, 1.6]]),
+            dict(type="ChromaticAutoContrast", p=0.2, blend_factor=None),
+            dict(type="ChromaticTranslation", p=0.95, ratio=0.1),
+            dict(type="ChromaticJitter", p=0.95, std=0.05),
+            # dict(type="HueSaturationTranslation", hue_max=0.2, saturation_max=0.2),
+            # dict(type="RandomColorDrop", p=0.2, color_augment=0.0),
+            dict(
+                type="GridSample",
+                grid_size=0.02,
+                hash_type="fnv",
+                mode="train",
+                return_grid_coord=True,
+            ),
+            dict(type="SphereCrop", sample_rate=0.8, mode="random"),
+            dict(type="NormalizeColor"),
+            dict(
+                type="InstanceParser",
+                segment_ignore_index=segment_ignore_index,
+                instance_ignore_index=-1,
+            ),
+            dict(type="ToTensor"),
+            dict(
+                type="Collect",
+                keys=(
+                    "coord",
+                    "grid_coord",
+                    "segment",
+                    "instance",
+                    "instance_centroid",
+                    "bbox",
+                ),
+                feat_keys=("color", "normal"),
+            ),
+        ],
+        test_mode=False,
+    ),
+    trainEval=dict(
```

Comment:
This is new data type that gets evaluated if the `use_eval_train` config option is passed to the `InsSegEvaluator`